Package.swift

```diff
@@ -22,21 +22,22 @@ let package = Package(
             path: "wire-runtime-swift/src/main/swift"
         ),
         // The tests depend on valid protos via gradle
-        // ./gradlew -p wire-library :wire-runtime-swift:generateTestProtos
+        // ./gradlew :wire-runtime-swift:generateTestProtos
```

Comment:
@oldergod 

> The comment makes it sound like the tests would not pass without having run generateSampleProtos but is that really the case? If not, no need for the comment I think ?

True, the samples _currently_ are not a dependency, and if they were we'd need the same codegen in the Cocoapods file. 

That said, I still would prefer we codegen the dinosaur sample in the repo somewhere.

Reply:
Got it